apps/sms/views/conversation/js/conversation.js

```diff
@@ -622,9 +618,9 @@ var ConversationView = {
         this.draft = Threads.get(threadId).getDraft();
 
         if (this.draft) {
-          this.draft.isEdited = false;
-
           Compose.fromDraft(this.draft);
+
+          this.draft.isEdited = false;
```

Comment:
That's one more question I have, do we need ```isEdited``` at all? Maybe we can save all the time without analyzing this flag?